magento/magento2
# Add EventPrefix and EventObject to checkout agreement model

### Description (*)

Added EvenPrefix and EventObject for checkout agreement model to trigger the event with event prefix.

### Related Pull Requests


### Fixed Issues (if relevant)

1. ...

### Manual testing scenarios (*)

1. Add observer for any of the following,
    * checkout_agreement_load_before
    * checkout_agreement_load_after
    * checkout_agreement_save_commit_after
    * checkout_agreement_save_before
    * checkout_agreement_save_after
    * checkout_agreement_delete_before
    * checkout_agreement_delete_after
    * checkout_agreement_delete_commit_after
    * checkout_agreement_clear
2. Check observer code is executed and also received 'agreement' in event data

### Questions or comments


### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#32510: Add EventPrefix and EventObject to checkout agreement model
